### PR TITLE
util/codec: add EncodedBytesLength

### PR DESCRIPTION
**Pull request by zhiqiangxu**
This helper function can be used in `encodeHashDataKey`, `encodeHashMetaKey` etc, to estimate the key size more accurately.

### CODE REVIEW

File: util/codec/bytes.go [hunk 1]
```diff
@@ -67,6 +67,14 @@ func EncodeBytes(b []byte, data []byte) []byte {
 	return result
 }
 
+// EncodedBytesLength returns the length of data after encoded
+func EncodedBytesLength(data []byte) int {
```
Comment (crazycs520) [inline util/codec/bytes.go]: Use `dataLen` as input parameters are better to use?

Comment (zhiqiangxu) [reply util/codec/bytes.go]: Sure, change commited :)

Comment (zhiqiangxu) [reply util/codec/bytes.go]: Is there something wrong with jenkins? After the last commit, test fails.

**Comment by zz-jason** [comment]
> This helper function can be used in `encodeHashDataKey`, `encodeHashMetaKey` etc, to estimate the key size more accurately.

@zhiqiangxu please also update these two places to use this function.

**Comment by zhiqiangxu** [comment]
> > This helper function can be used in `encodeHashDataKey`, `encodeHashMetaKey` etc, to estimate the key size more accurately.
> 
> @zhiqiangxu please also update these two places to use this function.

Applied to these two places.

**Pull request merged**